compiler/frontend/beam/src/main/java/org/apache/nemo/compiler/frontend/beam/transform/GroupByKeyTransform.java

```diff
@@ -26,53 +32,172 @@
 
 /**
  * Group Beam KVs.
- * @param <I> input type.
+ * @param <K> key type.
+ * @param <InputT> input type.
  */
-public final class GroupByKeyTransform<I> implements Transform<I, WindowedValue<KV<Object, List>>> {
+public final class GroupByKeyTransform<K, InputT>
```

Comment:
GroupByKeyDoFnTransform?
Since we're using `GroupAlsoByWindowViaWindowSetNewDoFn`.